tests/integration/test_controllers.py

```diff
@@ -532,7 +532,9 @@ def test_AIF_controller_with_simulated_chems(self):
 
         # create a simulated mass spec without noise and Top-N controller
         logger.info('Without noise')
-        mass_spec = IndependentMassSpectrometer(ionisation_mode, dataset, self.ps, add_noise=False, scan_duration_dict = scan_time_dict)
+        #mass_spec = IndependentMassSpectrometer(ionisation_mode, dataset, self.ps, add_noise=False, scan_duration_dict = scan_time_dict)
+        mass_spec = IndependentMassSpectrometer(ionisation_mode, dataset, self.ps, add_noise=False)
```

Comment:
why did the `scan_duration_dict` disappear from these? Doesn't really matter...

Reply:
just my own testing for using default with ms2 timing. will rollback to use scan_duration_dict.